### DISCUSSION TITLE
replacing space with +

### DISCUSSION THREAD

**Discussion by AstarothSquirrel** [microsoft-search/pnp-modern-search#2235] [Q&A]
There is probably a really simple way of doing this but I can't figure it.  I am using a pnp search box to send a query to our telephone directory.  However the search box sends spaces as %20 and I need to change this to "+"  e.g. currently "John Smith" gets sent as "John%20Smith" and what I need is "John+Smith"  I'm guessing that the "Query input transformation template" might be where I need to enter string operations but I can't find any documentation on how to do this.  TIA

**Comment by wobba** [comment]
Is your telephone directory using the PnP search result web part? If so decoding of the query should just work. If you have a custom web part, then you need to handle the decoding yourself as both %20 and + are valid as space from a query parameter.
